src/Components/main.css

```diff
@@ -1,3 +1,37 @@
+body{
+    font-family: Avenir;
+}
+
+.title_img{
+    background-image: url("/cryptography.jpg");
+}
+
+.title_design{
+    font-weight: bold;
+    font-size: 100px;
+    color: white;
+    padding-top: 200px;
+}
+
+.desc{
+    font-weight: normal;
+    font-size: 48px;
+    color: white;
+    padding: 50px;
+    width: 60%;
+    margin: auto;
+    position: relative;
+    text-align: center;
+    padding-bottom: 350px;
+}
+
+.content{
+    color: cipher-gold;
```

Comment:
I noticed you used a placeholder here for the content color property. Make sure this is defined!